### ISSUE TITLE
Sorting an array doesn’t compile when targeting nvvm

### ISSUE THREAD

**Issue by sebcrozet** [Rust-GPU/Rust-CUDA#11]
The following piece of code:

```rust
let mut foo = [1usize, 2];
foo.sort_unstable();
```

fails to compile with the error:

```
error: internal compiler error: C:\Users\devel\.cargo\registry\src\github.com-1ecc6299db9ec823\rustc_codegen_nvvm-0.1.0\src\context.rs:373:32: unknown intrinsic 'llvm.bswap.i64'
```

I originally triggered this error while attempting to use the sorted variant of the SVD decomposition in `nalgebra`.

**Comment by RDambrosio016** [comment]
Weird, looks like somewhere along the line i deleted bswap from ctx_intrinsics, which is wrong because nvvm [supports it](https://docs.nvidia.com/cuda/nvvm-ir-spec/index.html#bit-manipulations-intrinics)

**Comment by RDambrosio016** [comment]
Should be fixed in cg_nvvm 0.1.1
